.github/workflows/ci.yaml

```diff
@@ -81,9 +82,60 @@ jobs:
       - run: yarn build:internal-dependencies
       - run: ./packages/devops/scripts/ci/validateTypesAndDbSchemaInSync.sh
 
-  run_tests:
-    name: Run tests
+  validate_lockfile:
+    name: Validate lockfile
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.19.3
+          cache: yarn
+      - run: yarn set version 3.2.1
+      - run: SKIP_BUILD_INTERNAL_DEPENDENCIES=true yarn install --immutable
+
+  package_list:
```

Comment:
Not a core part of this PR but a nice little bit of automation so we don't have to maintain the package list.